cmd/syncIndexer.go

```diff
@@ -177,6 +178,63 @@ func (i *folderIndexer) store(storedObject StoredObject) (err error) {
 	return
 }
 
+// filesChangedInDirectory returns true in case any file in folder have ctime more than lastSyncTime.
```

Comment:
enhance comment

 	//
	// This MUST be called only with CFDMode==Ctime and MetadataOnlySync==true.
	// This forces caller (target traverser) to use the more efficient ListDir for
	// getting file properties in bulk instead of querying individual file properties.
	//